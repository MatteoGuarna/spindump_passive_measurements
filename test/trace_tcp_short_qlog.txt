Short wget TCP trace, no security. Same PCAP file as in
trace_tcp_short test case, but testing Qlog output from Spindump.

