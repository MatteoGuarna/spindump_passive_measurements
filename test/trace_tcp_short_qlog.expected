{"qlog_version": "draft-02", "qlog_format": "JSON", "description": "Spindump measurements", "traces": [
 { "vantage_point": { "type": "network" }, "events": [
     {"time": 1553418212984046, "protocol_type": TCP, "group_id": "sd-0", "event": "connection_started", 
      "ip_version": "ipv6", "src_ip": "2001:67c:1232:144:9498:6df6:f450:110b", "dst_ip": "2001:67c:2b0:1c1::198", "src_port": "59105", "dst_port": "80", 
      "data": {"packets1": 1, "packets2": 0, "bytes1": 84, "bytes2": 0, "bandwidth1": 84, "bandwidth2": 0}},
     {"time": 1553418213017044, "protocol_type": TCP, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 1, "packets2": 1, "bytes1": 84, "bytes2": 80, "bandwidth1": 84, "bandwidth2": 80, "right_rtt": 32998}},
     {"time": 1553418213017134, "protocol_type": TCP, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 1, "packets2": 1, "bytes1": 84, "bytes2": 80, "bandwidth1": 84, "bandwidth2": 80, "left_rtt": 90}},
     {"time": 1553418213061925, "protocol_type": TCP, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 3, "packets2": 1, "bytes1": 368, "bytes2": 80, "bandwidth1": 368, "bandwidth2": 80, "right_rtt": 44683}},
     {"time": 1553418213061986, "protocol_type": TCP, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 3, "packets2": 4, "bytes1": 368, "bytes2": 734, "bandwidth1": 368, "bandwidth2": 734, "left_rtt": 58}},
     {"time": 1553418213061987, "protocol_type": TCP, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 4, "packets2": 4, "bytes1": 440, "bytes2": 734, "bandwidth1": 440, "bandwidth2": 734, "left_rtt": 59}},
     {"time": 1553418213064120, "protocol_type": TCP, "group_id": "sd-1", "event": "connection_started", 
      "ip_version": "ipv6", "src_ip": "2001:67c:1232:144:9498:6df6:f450:110b", "dst_ip": "2001:67c:2b0:1c1::198", "src_port": "59106", "dst_port": "80", 
      "data": {"packets1": 1, "packets2": 0, "bytes1": 84, "bytes2": 0, "bandwidth1": 84, "bandwidth2": 0}},
     {"time": 1553418213098429, "protocol_type": TCP, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 6, "packets2": 4, "bytes1": 584, "bytes2": 734, "bandwidth1": 584, "bandwidth2": 734, "right_rtt": 34370}},
     {"time": 1553418213098432, "protocol_type": TCP, "group_id": "sd-1", "event": "measurement", 
      "data": {"packets1": 1, "packets2": 1, "bytes1": 84, "bytes2": 80, "bandwidth1": 84, "bandwidth2": 80, "right_rtt": 34312}},
     {"time": 1553418213098549, "protocol_type": TCP, "group_id": "sd-1", "event": "measurement", 
      "data": {"packets1": 1, "packets2": 1, "bytes1": 84, "bytes2": 80, "bandwidth1": 84, "bandwidth2": 80, "left_rtt": 117}},
     {"time": 1553418213134370, "protocol_type": TCP, "group_id": "sd-1", "event": "measurement", 
      "data": {"packets1": 3, "packets2": 1, "bytes1": 378, "bytes2": 80, "bandwidth1": 378, "bandwidth2": 80, "right_rtt": 35675}},
     {"time": 1553418213134415, "protocol_type": TCP, "group_id": "sd-1", "event": "measurement", 
      "data": {"packets1": 3, "packets2": 4, "bytes1": 378, "bytes2": 3152, "bandwidth1": 378, "bandwidth2": 3152, "left_rtt": 43}},
     {"time": 1553418213134659, "protocol_type": TCP, "group_id": "sd-1", "event": "measurement", 
      "data": {"packets1": 4, "packets2": 7, "bytes1": 450, "bytes2": 7652, "bandwidth1": 450, "bandwidth2": 7652, "left_rtt": 23}},
     {"time": 1553418213134805, "protocol_type": TCP, "group_id": "sd-1", "event": "measurement", 
      "data": {"packets1": 5, "packets2": 9, "bytes1": 522, "bytes2": 8254, "bandwidth1": 522, "bandwidth2": 8254, "left_rtt": 28}},
     {"time": 1553418213134806, "protocol_type": TCP, "group_id": "sd-1", "event": "measurement", 
      "data": {"packets1": 6, "packets2": 9, "bytes1": 594, "bytes2": 8254, "bandwidth1": 594, "bandwidth2": 8254, "left_rtt": 27}},
     {"time": 1553418213169945, "protocol_type": TCP, "group_id": "sd-1", "event": "measurement", 
      "data": {"packets1": 9, "packets2": 9, "bytes1": 810, "bytes2": 8254, "bandwidth1": 810, "bandwidth2": 8254, "right_rtt": 33074}}  
 ]}]}
